floodviz/views.py

```diff
@@ -1,7 +1,112 @@
 from flask import request, render_template, abort
+import csv
+import requests
+import yaml
+import os
 
 from . import app
 
+
 @app.route('/home/')
 def home():
-    return render_template('index.html')
\ No newline at end of file
+    return render_template('index.html')
+
+
+# This program requires svgis to be installed (which in turn requires Fiona and GDAL)
```

Comment:
I'm sorry that I led you down the wrong path here. I was hoping that all of our preprocessing steps would be within the view itself. The package svgis does have an API so it should be possible to create the SVG when the view is rendered.  I would like you to see if that is possible. Also did you look at using D3 to draw the map client side? Just wondering if you explored that.